src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.search__form {
+  display: flex;
+}
+
+.search__form-big {
+  margin-top: 20px;
+}
+
+.search__form-small {
+  margin-top: 20px;
+}
+
+.input {
+  display: flex;
+  outline: 0;
+  padding: 0;
+
+  width: 100vw;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  height: 68px;
+  background-size: 19px;
+  background-position: 26px;
+  padding-left: 62px;
```

Comment:
<img width="1421" alt="Screenshot 2023-08-02 at 18 39 43" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/53228259/7c35ab65-e282-4f27-8301-30502d1450df">
check design you failed tests

Reply:
not fixed